engine/core/pom.xml

```diff
@@ -81,6 +81,17 @@
 			<artifactId>asm5</artifactId>
 		</dependency>
 
+        <dependency>
+            <groupId>com.fasterxml.jackson.module</groupId>
+            <artifactId>jackson-module-jsonSchema</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.eobjects.datacleaner</groupId>
+            <artifactId>DataCleaner-rest-client</artifactId>
```

Comment:
No, rest-client is very simple client, not using any engine classes. Actually it should not depend on DC api (which I will repair in some next commit). On the other hand, the engine-core now has the remote transformer which uses the rest-client to call the server.